core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -142,15 +143,15 @@ public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse
      * The next poll cycle will be defined by retryAfter value in {@link PollResponse}.
      * In absence of {@link PollResponse#getRetryAfter()}, the {@link Poller} will use {@code pollInterval}.
      *
-     * @param pollInterval Not-null and greater than zero poll interval.
-     * @param pollOperation The polling operation to be called by the {@link Poller} instance. This is a callback into the client library,
-     * which must never return {@code null}, and which must always have a non-null {@link com.azure.core.util.polling.PollResponse.OperationStatus}.
-     *{@link Mono} returned from poll operation should never return {@link Mono#error(Throwable)}.If any unexpected scenario happens in poll operation,
+     * @param pollInterval    Not-null and greater than zero poll interval.
+     * @param pollOperation   The polling operation to be called by the {@link Poller} instance. This is a callback into the client library,
```

Comment:
Don't do this kind of alignment

Reply:
I  have auto format in Intellij which I should turn off.